### ISSUE TITLE
Improve the LoKi functors lesson

### ISSUE THREAD

**Issue by vlisovsk** [lhcb/starterkit-lessons#38]
From Vanya:

> In [LoKi functors lesson](https://lhcb.github.io/starterkit-lessons/first-analysis-steps/loki-functors.html), one probably can avoid a long paragraph with discussions on VFASPF, there are functors CHI2VX and CHI2VXNDOF that get particle as an argument and evaluate corresponding chi2_VX ( or chi2_VX/ndf).
> I think it could simplify a bit the material.

Though, we would like to keep the `VFASPF` discussion since it's widely used in the stripping lines. Just a brief update on the `CHI2VX` and `CHI2VXNDOF` should be useful.

**Comment by alexpearce** [comment]
I agree. Another reason for keeping `VFASPF` is that it shows how functors can return functors.

**Comment by apuignav** [comment]
+1
